pyproject.toml

```diff
@@ -108,7 +107,7 @@ disallow_any_generics = true
 # site-packages is here to help vscode mypy integration getting confused
 exclude = "(build|dist|test/local-content|site-packages|~/.pyenv|examples/playbooks/collections|plugins/modules)"
 # https://github.com/python/mypy/issues/12664
-no_incremental = true
+incremental = false
```

Comment:
@ssbarnea please confirm if this is okay, as for `no_incremental` we're getting an error `Additional properties are not allowed ('no_incremental' was unexpected)`